reproducible_build.sh

```diff
@@ -357,6 +368,13 @@ function build_lbrycrd() {
     strip src/test/test_lbrycrd
 }
 
+function clang_format_diff(){
+    # run a code formatting check on any commits not in master
+    # requires clang-format
+    git fetch origin master
+    git diff -U0 origin/master -- '*.h' '*.cpp' | ./contrib/devtools/clang-format-diff.py -p1
```

Comment:
I don't think this will do what you expect it to do.

Try:

```
git remote add origin2 https://github.com/lbryio/lbrycrd.git
git fetch origin2
git diff -U0 origin2/master -- ... snip ...
```